lib/logic/models/sort.dart

```diff
@@ -1,21 +1,25 @@
 import 'package:enum_to_string/enum_to_string.dart';
 import 'package:equatable/equatable.dart';
-import 'package:flutter/foundation.dart';
 import 'package:sweyer/sweyer.dart';
 
 /// Interface for other sort feature enums.
 abstract class SortFeature<T extends Content> extends Enum<String> {
```

Comment:
This could be refactored using enhanced enums as well, by the way. But let's keep this for another patch.

Reply:
Yeah, one thing at a time. 😄